content/knowledge_base/consensus/avalanche/specification.md

```diff
@@ -0,0 +1,45 @@
+# Avalanche
+
+An internet-scale electronic payment system, evaluated in a large scale deployment, constructed with Snow family of algorithms. Avalance's tested confirmation latency goes down to 1.35 sec. System's bottleneck is transaction verification.
+TODO: elaborate more on the introduction
+
+
+## Characteristics
+
+Avalanche network operates with multiple single-decree instances (nodes), meaning that each instance can make a decision on only one value. Nodes use Snowball algorithm for the process of making a decision. (TODO: add a reference) In the case of a payment system and Avalanche, the Snowball logic used to determine a value is a process of deciding on a transaction being valid or not. 
```

Comment:
```suggestion
The Avalanche network operates with multiple single-decree instances (nodes), meaning that each instance can make a decision on only one value. Nodes use the Snowball algorithm for their decision-making process. In the case of a payment system and Avalanche, the Snowball logic used to determine a value is the process of deciding whether a transaction is valid. 
```

IMO the references within sentences should be included in this PR, as it also makes it easier to review properly and we might forget them in the future otherwise since they're hidden within paragraphs. WDYT?

Reply:
Yes, I'll go over them and add them, thanks!

Reply:
This text has been updated, added references as well
https://github.com/optakt/optakt.github.io/pull/13/commits/0a1cb902f3417c3bc20f9197df71d3d65a1b6421#diff-69f45991c0732c6f5ff297a5a86ffa91662d50369fc8057b3875361f5b80a2fcR10-R16
Thanks!